samples/bluetooth/mesh/light_ctrl/README.rst

```diff
@@ -134,6 +136,13 @@ FEM support
 
 .. include:: /includes/sample_fem_support.txt
 
+Emergency data storage
+======================
+To build this sample with support for emergency data storage, set ``OVERLAY_CONFIG`` to :file:`overlay-emds.conf`. This will save RPL data and some of the :ref:`bt_mesh_lightness_srv_readme` data to the emergency data storage instead of to the :ref:`settings_api`.
```

Comment:
Probably it is worth specifying which exact data will be stored with emds in the lightness model if this has not been specified anywhere.

Reply:
That should be described in the lightness model and not here.